server/graphql/loaders/collective.ts

```diff
@@ -200,16 +200,18 @@ export default {
           const results: Collective[] = await sequelize.query(
             `
           SELECT
-            fc.*, JSONB_OBJECT_AGG(cas."CollectiveId", cas."relations") AS "associatedCollectives",
-            cas."HostCollectiveId" AS "contextualHostCollectiveId", MAX(cas."lastInteractionAt") as "lastInteractionAt", MIN(cas."firstInteractionAt") as "firstInteractionAt" 
+            fc.*,
+            JSONB_OBJECT_AGG(cas."CollectiveId", cas."relations") FILTER (WHERE c."type" IN ('COLLECTIVE', 'FUND', 'PROJECT', 'EVENT')) AS "associatedCollectives",
+            JSONB_OBJECT_AGG(cas."CollectiveId", cas."relations") FILTER (WHERE c."type" = 'ORGANIZATION') AS "associatedOrganizations",
```

Comment:
Turns out they were already returned among the list of collectives. I've separated them into a new `associatedOrganizations` field.